app/components/site-chrome/template.hbs

```diff
@@ -8,12 +8,12 @@
   <div class="sitechrome-top">
     <div class="sitechrome-topbody">
       <div class="sitechrome-topbody-right">
-          {{#if media.isLargeAndUp}}
+        <!-- if media is isLargeAndUp -->
```

Comment:
Is this managed with a media query?

Reply:
whoops, forgot to take this out. yes, this is now managed w/ a media query in css.